src/Apps/Legion/Components/StepperWizard/Internal/Step.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { FontIcon } from '@fluentui/react';
+import { IStepProps } from './Step.types';
+
+export const Step: React.FC<IStepProps> = ({
+    type,
+    label,
+    onClick,
+    isSelected,
+    isFinished,
+    isNavigationDisabled,
+    includeIcons,
+    isAllCompletedSuccessfully,
+    hasWarning
+}) => {
+    return (
```

Comment:
don't love this stepper wizard component. I know that it's carry forward from existing components but there's definitely some older patterns here like the SCSS and clickable divs

Reply:
I could take some time to update it to newer patterns, just wanted to have the structure and functionality ready as P0, and updates of the component as P1.